java/src/jmri/implementation/configurexml/MergSD2SignalHeadXml.java

```diff
@@ -136,14 +126,16 @@ public boolean load(Element shared, Element perNode) {
                 home = false;
             }
         }
-        try {
-            aspects = shared.getAttribute("aspects").getIntValue();
-        } catch (org.jdom2.DataConversionException e) {
-            log.warn("Could not parse level attribute!");
-        } catch (NullPointerException e) {  // considered normal if the attribute not present
+
+        var aspectsAttr = shared.getAttribute("aspects");
+        if ( aspectsAttr != null ) { // considered normal if the attribute not present
```

Comment:
The comment looks wrong to me.
... if the attribute is _not_ present

Reply:
So I think the comment should be removed.

Reply:
Aah, sure, see latest tweak.